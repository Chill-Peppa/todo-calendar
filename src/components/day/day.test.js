import { render } from '@testing-library/react';
import Day from './day';

it('should correctly render Day', () => {
  const item = { day: 15, isHoliday: 0 };
  const index = 0;
  const onOpenTodo = jest.fn();
  const currentMonth = 9;
  const currentYear = 2023;
  const checkDate = (value) => value === '2023-10-15';

  render(
    <Day
      item={item}
      index={index}
      onOpenTodo={onOpenTodo}
      currentMonth={currentMonth}
      currentYear={currentYear}
      checkDate={checkDate}
    />,
  );

  expect(Day).toMatchSnapshot();
});
