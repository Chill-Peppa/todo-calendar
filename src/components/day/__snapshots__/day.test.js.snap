// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Day 1`] = `[Function]`;
